public/app/features/plugins/PluginsErrorsInfo.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { selectors } from '@grafana/e2e-selectors';
+import { HorizontalGroup, InfoBox, List, useTheme } from '@grafana/ui';
+import { mapPluginErrorCodeToSignatureStatus, PluginSignatureBadge } from './PluginSignatureBadge';
+import { StoreState } from '../../types';
+import { getAllPluginsErrors } from './state/selectors';
+import { loadPlugins, loadPluginsErrors } from './state/actions';
+import useAsync from 'react-use/lib/useAsync';
+import { connect, MapDispatchToProps, MapStateToProps } from 'react-redux';
+import { hot } from 'react-hot-loader';
+import { PluginError } from '@grafana/data';
+import { css } from 'emotion';
+
+interface ConnectedProps {
+  errors: PluginError[];
+}
+
+interface DispatchProps {
+  loadPluginsErrors: typeof loadPluginsErrors;
+}
+
+interface OwnProps {
+  children?: React.ReactNode;
+}
+
+type PluginsErrorsInfoProps = ConnectedProps & DispatchProps & OwnProps;
+
+export const PluginsErrorsInfoUnconnected: React.FC<PluginsErrorsInfoProps> = ({
+  loadPluginsErrors,
+  errors,
+  children,
+}) => {
+  const theme = useTheme();
+
+  const { loading } = useAsync(async () => {
+    await loadPluginsErrors();
+  }, [loadPlugins]);
+
+  if (loading || errors.length === 0) {
+    return null;
+  }
+  return (
+    <InfoBox
+      aria-label={selectors.pages.PluginsList.signatureErrorNotice}
+      severity="warning"
+      urlTitle="Read more about plugins signing"
```

Comment:
```suggestion
      urlTitle="Read more about plugin signing"
```

making this singular reads a bit better